drivers/volume/portworx/mock/ops.go

```diff
@@ -0,0 +1,465 @@
+// +build unittest
```

Comment:
You don't need this, you can use the fake client. Look at the storkctl unit tests for an example